lib/utils.js

```diff
@@ -279,6 +279,51 @@ function extend(a, b, thisArg) {
   return a;
 }
 
+function mergeConfig(defaults, instanceConfig) {
+  var config = {};
+  forEach(['url', 'method', 'params', 'data'], function(prop) {
```

Comment:
modify this line to `... function assignProp(prop)` to pass eslint errors in CI tests (or some other function name of your choosing)